app/src/main/java/com/edulectronics/tinycircuit/DataStorage/VariableHandler.java

```diff
@@ -0,0 +1,32 @@
+package com.edulectronics.tinycircuit.DataStorage;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+
+/**
+ * Created by Wilmer on 11-1-2017.
+ */
+
+public class VariableHandler {
+    private SharedPreferences sharedPreferences;
+    private SharedPreferences.Editor editor;
+
+    //Setup for save file
+    public VariableHandler(Context context) {
+        sharedPreferences = PreferenceManager.getDefaultSharedPreferences(context);
```

Comment:
wat zijn shared Preferences? Zijn er ook niet-shared preferences? En wat voor variables worden hier gehandles? Als het niet duidelijk te maken is door naamgeving, dan graag comments :)